src/App.css

```diff
@@ -69,6 +69,15 @@ body {
   display: inline-block !important;
 }
 
+.label-inline {
+  padding-right: 5px !important;
+  display: inline-block !important;
+}
+
+.text-red {
```

Comment:
It might be better to override the required text directly rather than imitating the styling.
You could override this selector to set the content:
```
.ui.form .required.field > label::after {
  content: '(required)';
}
```